core/src/main/java/org/stellar/anchor/util/Sep1Helper.java

```diff
@@ -10,7 +10,13 @@ public static TomlContent readToml(String url) throws IOException {
   }
 
   public static TomlContent parse(String tomlString) {
-    return new TomlContent(tomlString);
+    try {
+      return new TomlContent(tomlString);
+    } catch (Exception e) {
+      // obfuscate exception message to prevent metadata leaks
```

Comment:
Should we still log it on ERR level? Otherwise it's hard to debug what went wrong